aiosmtpd/smtp.py

```diff
@@ -444,9 +452,7 @@ def smtp_MAIL(self, arg):
                 return
         if self.enable_SMTPUTF8:
             smtputf8 = params.pop('SMTPUTF8', False)
-            if smtputf8 is True:
-                self.require_SMTPUTF8 = True
-            elif smtputf8 is not False:
+            if (smtputf8 is not True) and (smtputf8 is not False):
```

Comment:
I spend a little time to think about that parameters once more. Looks like they are really useless by their meaining. We can`t believe client encoding declaration. It may be middleware MTA in chain of different servers with different encoding settings. If 8bit encoding and utf-8 support servers are mixed in chain, everything could happen. So best idea is just everytime use 'utf-8 surrogateescape' and make end user decide if it wants to deal with encoding problems. In most cases, I think, noone would care, if email can be passed to next layer of application without problems.